### ISSUE TITLE
vgg_normalized.pth vs torchvision pretrained vgg

### ISSUE THREAD

**Issue by ZeroAct** [naoto0804/pytorch-AdaIN#43]
Hi.

I wonder the difference between 'vgg_normalized.pth' you uploaded and pretrained vgg19 from torchvision.

I tried to train AdaIN with pretrained vgg19 from torchvision many times, but I've failed.
(I've succeed with 'vgg_normalized.pth')

Sample of my AdaIN inferences that trained with torchvision vgg19 for so many epochs looking so bad...
![epoch_761](https://user-images.githubusercontent.com/46982469/129296766-fc9a7e58-ac04-4d50-989c-44159fb11d2d.png)

Thank you : )

**Comment by wjjjjyourFA** [comment]
Hi, may I know how you did it？ 
I put a image in ./input/content and another in ./input/style, using "vgg_normalized.pth"
CMD:`python train.py --content_dir input/content --style_dir input/style --batch_size 4 --max_iter 160`
Though I got a xx.pth.tar in ./experiments, I use it "test.py", getting a gray image result.

**Comment by naoto0804** [comment]
@wjjjjyourFA 
I guess `max_iter` is too small. Please try the default parameters, or keep an eye on the convergence of loss curves.

**Comment by naoto0804** [comment]
@ZeroAct I guess the VGG model provided is a bit modified. I am still not sure what the `_normalized` actually means.

**Comment by ZeroAct** [comment]
@wjjjjyourFA I wrote my new AdaIN code. So, I can't answer you. Sorry!

**Comment by ZeroAct** [comment]
@naoto0804 Thank you for your answer. It was very helpful.
